src/generate.ts

```diff
@@ -19,8 +19,8 @@ export const csvGenerate = (
   const headerKeys = Object.keys(data[0]);
   const columnNames = headers ?? headerKeys;
   const csv = data.map((row) => csvGenerateRow(row, headerKeys, delimiter));
-
   csv.unshift(columnNames.join(delimiter));
+  csv.unshift(`sep=${delimiter}`);
```

Comment:
Left a comment about compatibility in #50

Reply:
I think a better solution to this problem would be to change the default delimiter to be CSV, to comply with RFC-4180: https://github.com/coston/json-to-csv-export/issues/61